screenshot.html

```diff
@@ -250,11 +251,14 @@ <h3 id=screenshot-header></h3>
       const {url, hash, ext, type, dst} = parseQuery(decodeURIComponent(window.location.search));
 
       try {
-        const _loadGltf = async () => {
+        const _loadGltf = async (passedUrl) => {
+          if (!passedUrl) {
```

Comment:
Why would this happen? If some file is missing this doesn't seem like the place to make one up.

Reply:
I would move this defaulting outside of this function or drop it altogether.

Reply:
droped it